<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Heap Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="arrayHeapGroup">
      <property name="title">
       <string>Array Heap</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutArray">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutArrayInput">
         <item>
          <widget class="QLineEdit" name="arrayInput">
           <property name="placeholderText">
            <string>Enter value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="insertArrayButton">
           <property name="text">
            <string>Insert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="extractArrayButton">
           <property name="text">
            <string>Extract Max</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="arrayDisplay">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="listHeapGroup">
      <property name="title">
       <string>List Heap</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutList">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutListInput">
         <item>
          <widget class="QLineEdit" name="listInput">
           <property name="placeholderText">
            <string>Enter value</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="insertListButton">
           <property name="text">
            <string>Insert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="extractListButton">
           <property name="text">
            <string>Extract Max</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="listDisplay">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
